lib/LaTeXML/Package/mws.sty.ltxml

```diff
@@ -70,18 +70,18 @@ DefMathML('Apply:?:range',
         # get it
         my $low = $_[1]->textContent;
         my $high = $_[2]->textContent;
-        if($low && $high){
```

Comment:
the cleaner way to do this in perl is
```perl
if (defined $low && defined $high)
```

you wouldn't have to change any of the other lines then